apps/web/package.json

```diff
@@ -24,21 +24,23 @@
     "@lucia-auth/adapter-drizzle": "^1.0.7",
     "@monaco-editor/react": "^4.6.0",
     "@t3-oss/env-nextjs": "^0.10.1",
+    "ai": "^3.2.42",
     "bcrypt": "^5.1.1",
     "bullmq": "^5.8.5",
     "ioredis": "^5.4.1",
     "lodash-es": "^4.17.21",
     "lucia": "^3.2.0",
     "monaco-editor": "^0.50.0",
     "next": "^14.3.0-canary.87",
+    "next-themes": "^0.3.0",
```

Comment:
?

Reply:
after rebasing this dependency was missing